content/ros/services/services_actions.tex

```diff
@@ -0,0 +1,156 @@
+\documentclass{beamer}
+
+
+\usepackage[utf8]{inputenc}
+\usepackage{amsmath}
+\usepackage{amsfonts}
+\usepackage{amssymb}
+\usepackage{graphicx}
+\usepackage{ragged2e}  % `\justifying` text
+\usepackage{booktabs}  % Tables
+\usepackage{tabularx}
+\usepackage{tikz}      % Diagrams
+\usetikzlibrary{calc, shapes, backgrounds}
+\usepackage{amsmath}
+\usepackage{amssymb}
+\usepackage{listings}  % Code listings
+\usepackage[T1]{fontenc}
+\usepackage{url}       % `\url
+\usepackage{dsfont}
+
+
+
+\usepackage{theme/beamerthemehbrs}
+
+\author[Kramer]{Erick Kramer}
+\title{Introduction to service and client ROS}
+\subtitle{}
+\institute[HBRS]{Hochschule Bonn-Rhein-Sieg}
+\date{04.04.2018}
+\subject{Test beamer}
+
+% \thirdpartylogo{path/to/your/image}
+
+
+\begin{document}
+{
+\begin{frame}
+\titlepage
+\end{frame}
+}
+
+\AtBeginSection[]{% Print an outline at the beginning of sections
+\begin{frame}<beamer>
+	\tableofcontents[currentsection]
+\end{frame}%
+}%
+
+\section{Services (Theory)}
+	\begin{frame}
+		\frametitle{What is a Service?}
+		\begin{itemize}
+			\item{It is a request/reply communication paradigm.}
+			\item{It is composed of two pair of messages, one for the request and one for the reply.}
+			\item{Two nodes participate in the communication process.}
+			\item{One node offers the service and other node request the service and wait for the response.}
+			\item{A service is defined using \textbf{srv} files}
+			
+		\end{itemize}
+	\end{frame}
+	\begin{frame}
+		\frametitle{Service type}
+		\begin{itemize}
+			\item{The service type is the package resource name of the .srv file, i.e. package name + name of the .srv file.}
+			\item{E.g. mysrvs/srv/PolledImage.srv has the service type mysrvs/PolledImage }
```

Comment:
you can use \texttt to highlight the file pahts+names and the service type/service name, so that it looks like this:
`mysrvs/srv/PolledImage.srv` and `mysrvs/PolledImage`